.travis.yml

```diff
@@ -14,3 +14,25 @@ script:
 - "./debuild.sh |& tee log.txt"
 notifications: 
 slack: fossasia:3pMkQrSogvEMfnu9aJDW2ZYC
+env:
+  global:
+  - PACKAGE_NAME=meilix-systemlock
```

Comment:
@NishithP2004 The travis env is enabled but the package name is systemlock_0.1 not the meilix-systemlock